Mathlib/Analysis/BoxIntegral/UnitPartition.lean

```diff
@@ -293,4 +308,183 @@ theorem prepartition_isPartition {B : Box ι} (hB : hasIntegralVertices B) :
   rw [TaggedPrepartition.mem_toPrepartition, mem_prepartition_iff]
   exact ⟨index n x, mem_admissibleIndex_of_mem_box n hB hx, rfl⟩
 
+theorem exists_admissibleIndex {B : Box ι} (hB : hasIntegralVertices B) {x : ι → ℝ} (hx : x ∈ B) :
+    ∃ ν ∈ admissibleIndex n B, box n ν = box n (index n x) :=
+  ⟨index n x, mem_admissibleIndex_of_mem_box n hB hx, rfl⟩
```

Comment:
Doesn't this just say `index n x ∈ admissibleIndex n B` (which is `mem_admissibleIndex_of_mem_box n hB hx`)? Why this more convoluted statement?
(I guess the actual question is, can this not easily be inlined where it is used?)

Reply:
Indeed, it can be easily inlined at the only place it is used.